data/campaigns/Legend_of_Wesmere/utils/low-macros.cfg

```diff
@@ -59,13 +59,16 @@
 #enddef
 
 #define ELVES_SCOUTS
-Elvish Scout, Elvish Rider, Elvish Outrider, Elvish Horse Archer#enddef
+Elvish Scout, Elvish Rider, Elvish Outrider
+#enddef
 
 #define ELVES_STEEL
-Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}#enddef
+Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}
+#enddef
 
 #define ELVES_NON_STEEL
-Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}#enddef
+Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}
+#enddef
```

Comment:
Yeah, I have very little idea how WML works, and I don't want to be accountable in any way if the code breaks. I think it's better to fix these things all at once or not at all.